src/lib/build-actions.js

```diff
@@ -21,10 +21,10 @@ const { buildActions } = require('@adobe/aio-lib-runtime')
  * @param {boolean} [forceBuild=false] force a build (skip file changed hash check)
  */
 module.exports = async (config, filterActions, forceBuild = false) => {
-  utils.runScript(config.hooks['pre-app-build'])
-  const script = await utils.runScript(config.hooks['build-actions'])
+  runInProcess(config.hooks['pre-app-build'], config)
```

Comment:
do we need to await here, if not there is a possibility that `build-actions` is run before `pre-app-build` runs? Similarly for `post-app-build`

Reply:
Fixed